Pages.py

```diff
@@ -1,873 +1,1220 @@
+        self.state.current_user.sms = not self.state.current_user.sms
+        if (self.state.save_accounts() == True):
+            print("\nPermission changed Successfully")
+            self.input_to_continue()
+
+    def change_ad_permission(self):
+        self.state.current_user.targeted_advertising = not self.state.current_user.targeted_advertising
+        if (self.state.save_accounts() == True):
+            print("\nPermission changed Successfully")
+            self.input_to_continue()
+
+
+class CookiePolicy(Page):
+    def print_content(self):
+        print(f"{self.split_tilde}\n This is Cookie Policy Page.\n")
+
+    def onLoad(self):
+        clear_console()
+        self.print_content()
+        self.navigate()
+        pass
+
+    def navigate(self):
+        self.print_menu()
+        self.page_select()
+
+
+class CopyrightPolicy(Page):
+    def print_content(self):
+        print(f"{self.split_tilde}\n This is Copyright Policy Page\n")
+
+    def onLoad(self):
+        clear_console()
+        self.print_content()
+        self.navigate()
+        pass
+
+    def navigate(self):
+        self.print_menu()
+        self.page_select()
+
+
+class BrandPolicy(Page):
+    def print_content(self):
+        print(f"{self.split_tilde}\n This is Brand Policy Page.\n")
+
+    def onLoad(self):
+        clear_console()
+        self.print_content()
+        self.navigate()
+        pass
+
+    def navigate(self):
+        self.print_menu()
+        self.page_select()
+
+
+class Language(Page):
+    def print_content(self):
+        print(f"Manage your InCollege Account!\n{self.split_star}")
+        print(
+            f"\tUsername: {self.state.current_user.username}\n{self.split_tilde}")
+
+    def onLoad(self):
+        clear_console()
+        self.print_content()
+        self.menu()
+        pass
+
+    def navigate(self):
+        self.print_menu()
+        self.page_select()
+
+    def menu(self):
+        print("Select an option to change on your account or return home:\n")
+        print(f"\t0. Return Home")
+        print(
+            f"\t1. Update Language (Currently: {self.state.current_user.language})")
+        selection = input(
+            "Type your preferred option or enter corresponding number: ")
+        selection = "".join(selection.split()).lower()
+        if (selection == "langauge" or
+            selection == "1" or
+                selection == "1."):
+            self.change_language()
+        elif (selection == "home" or
+              selection == "0" or
+              selection == "0." or
+              selection == "returnhome" or
+              selection == "return"):
+            self.state.current_page = self.parent
+
+    def change_language(self):
+        new_language = input("\nEnter 0 for English, 1 for Spanish: ")
+        while (new_language == ""):
+            new_language = input("\nEnter 0 for English, 1 for Spanish: ")
+        self.state.current_user.language = new_language

```

Comment:
There should be input sanitation and checking here; there's nothing stopping a user from entering some arbitrary string as input for the new language which could break the program.